worker/remoterelations/doc.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package remoterelations contains workers which manage the operation of cross model relations.
+// The workers run on both the consuming and offering models.
```

Comment:
Which workers run where? Is it one worker on consuming, one worker on offering, or both workers on both sides?

Reply:
Added an extra bit of clarification